includes/class-wc-payments.php

```diff
@@ -524,11 +524,7 @@ public static function init() {
 				self::$upe_payment_gateway_map[ $payment_method->get_id() ] = $split_gateway;
 			}
 
-			self::$card_gateway = self::get_payment_gateway_by_id( 'card' );
-
-			$card_payments_checkout = new WC_Payments_Checkout( self::$legacy_card_gateway, self::$woopay_util, self::$account, self::$customer_service );
-			$card_payments_checkout->init_hooks();
```

Comment:
I'm a little nervous about removing these pieces and hooks...but all they do AFAICT is enqueue the scripts/DOM elements for the legacy card gateway, which should no longer be required, since the deferred intent UPE uses UPE elements for all gateways now.

Reply:
This makes sense to me given that legacy card gateway will no longer be used here under any condition. Looks like we're getting all the scripts we need for the deferred intent in the `init_hooks` call [here](https://github.com/Automattic/woocommerce-payments/pull/7058/files#diff-57ee486c4a3ba4bae478eb31255c9a968b5b4eccd04b27f512e4003440d6f821R543). I think it's safe 👍.